src/Application.php

```diff
@@ -639,33 +468,15 @@ private function checkForDuplicateRoute($path, $methods = null)
      */
     private function emitMarshalServerRequestException($exception)
     {
-        $response = (new Response())
-            ->withStatus(StatusCode::STATUS_BAD_REQUEST);
-        $finalHandler = $this->getFinalHandler();
-        $response = $finalHandler(new ServerRequest(), $response, $exception);
+        if ($this->container && $this->container->has(Middleware\ErrorResponseGenerator::class)) {
```

Comment:
I have to admit I'm not a huge fan of this. It's basically bypassing any `ErrorHandler` class pipeline that is already in the system - specifically the hooks into listeners. For instance, my db and APM error logging is done through an `ErrorLoggerListener` class that is attached to the `ErrorHandler` via a factory.